.rubocop.yml

```diff
@@ -12,6 +12,10 @@ Layout/EmptyLineBetweenDefs:
   Enabled: true
 Layout/IndentationConsistency:
   Enabled: true
+Metrics/MethodLength:
+  Exclude:
+    - "app/controllers/families_controller.rb"
```

Comment:
This should not be here. Which method is it complaining about? and what is rubicon output?

Reply:
It's complaining about the family_params that is too long and I was not sure how to  fix it. Any suggestion for a better way please?

```def family_params
    params.require(:family).permit(
      :first_name,
      :last_name,
      :address,
      :animals,
      :capability,
      :preferred_gender,
      :prof,
      :bus_driver,
      :active
    )
  end```

Reply:
Maybe change the limit for method length?